app/get_instances.gradle.kts

```diff
@@ -0,0 +1,103 @@
+import java.io.OutputStreamWriter
```

Comment:
For generating a static list like this, you can use whatever you like, rust, kotlin, bash, node, anything. All that matters is that it replaces those lines in the default_instances, so its *results* are compiled.

I'll only run it before releases anyway, and double check its results. Just make sure you give instructions on how to run it in the README.

Reply:
Since it's already in Kotlin I might as well leave as is, it's nice that it's integrated into gradle. I did notice that the AndroidManifest needs the references to the domains too, should I update that too?

Reply:
I wouldn't worry about it for this script. I'm going to remove all deeplinking at some point as its extremely complicated for federated apps.

Reply:
Ah fair enough